test/matrices.jl

```diff
@@ -0,0 +1,21 @@
+@testset "Matrices" begin
+  # tests with Aitchson's definitions
+  d = 10
+  D = d+1
+  @test J(d) == ones(d, d)
+  @test F(d) == [I(d) -ones(d)]
+  @test norm(G(D) - I(D)  + J(D)/ D, Inf) < 1e-5
+  @test H(d) == I(d) + J(d)
```

Comment:
These are great, they are operating directly on the FillArrays.jl instances. We also need tests to check that multiplication with J F H G directly works without specifying the dimension d.

Reply:
The tests without `d` are already there, for Matrix x Vector. The tests of type Matrix x Matrix are also ready, in the first commit.

Reply:
https://github.com/JuliaEarth/CoDa.jl/blob/fe6b179cb5d4c3c47ad3e59d8609f5d020273742/test/matrices.jl#L10-L20

Reply:
these tests, right?

Reply:
Maybe the diff on GitHub is outdated? I can't see the tests with the dimensionless matrices.

Reply:
Exactly 👌🏽 Tests like this one. We need similar tests for the J, G, H matrices.

Reply:
they were already there in the first commit, maybe that's the problem

Reply:
M is a loop for the square matrices J, G and H